desktop/apps/artsy_primer/personalize/stylesheets/favorites.styl

```diff
@@ -3,6 +3,15 @@
   max-width 1080px
   &[data-step='favorites']
     width 80%
+    @media screen and (max-width: 768px)
```

Comment:
Maybe we should make a stylus mixin for these, `768px` could be something easily forgettable.